go/runtime/client/client.go

```diff
@@ -137,6 +148,15 @@ func (c *runtimeClient) submitTx(ctx context.Context, request *api.SubmitTxReque
 
 // Implements api.RuntimeClient.
 func (c *runtimeClient) SubmitTx(ctx context.Context, request *api.SubmitTxRequest) ([]byte, error) {
+	resp, err := c.SubmitTxMeta(ctx, request)
```

Comment:
Does this now ignore errors during CheckTx?

Reply:
Yeah it did, fixed. Will update the mock runtime used in node tests to enable forcing check-tx errors so this can be tested.